### ISSUE TITLE
Commercial usage

### ISSUE THREAD

**Issue by jaideep11061982** [Deci-AI/super-gradients#983]
Is this model available for commercial use ?
https://learnopencv.com/yolo-nas/  This portal mentions non Comm but license Apache 2 says commercial.

**Comment by jaideep11061982** [comment]
@BloodAxe @ofrimasad @jonathan-sha @spsancti

**Comment by spsancti** [comment]
Hi!
You can use SuperGradients' source code to train your own YOLO-NAS **from scratch** for commercial use, as SuperGradients is under Apache license. 
You cannot, however, use our pretrained YOLO-NAS weights for commercial use as they are under a different license. 

https://github.com/Deci-AI/super-gradients/issues/901

**Comment by Louis-Dupont** [comment]
*Note that besides YOLO-NAS, our other pre-trained models (`YoloX`, `PPYoloE`, ... ) are available for commercial use (see [model zoo](https://docs.deci.ai/super-gradients/documentation/source/model_zoo.html))

**Comment by NeovMs** [comment]
If we use the pre-trained models for knowledge distillation to pre-train the Yolo-NAS from scratch, we're not strictly using the weights. In this case, can we get round the licence ?

**Comment by BloodAxe** [comment]
@NeovMs this is not a legal advice, you should talk to real lawyers, but for my perspective KD still uses pretrained weights and this training regime still falls under license listed here.
